### ISSUE TITLE
no compatible version found

### ISSUE THREAD

**Issue by winfinit** [npm/npm#9887]
```
root@ubuntu:/home/rjurkov# npm install -g mongodb-prebuilt --verbose
npm info it worked if it ends with ok
npm verb cli [ '/usr/local/bin/node',
npm verb cli   '/usr/local/bin/npm',
npm verb cli   'install',
npm verb cli   '-g',
npm verb cli   'mongodb-prebuilt',
npm verb cli   '--verbose' ]
npm info using npm@2.11.2
npm info using node@v0.12.6
npm verb node symlink /usr/local/bin/node
npm verb install initial load of /usr/local/lib/package.json
npm verb readDependencies loading dependencies from /usr/local/lib/package.json
npm verb cache add spec mongodb-prebuilt
npm verb addNamed "*" is a valid semver range for mongodb-prebuilt
npm verb addNameRange registry:https://registry.npmjs.org/mongodb-prebuilt not in flight; fetching
npm verb request uri https://registry.npmjs.org/mongodb-prebuilt
npm verb request no auth needed
npm info attempt registry request try #1 at 4:19:34 PM
npm verb request id 3288754daae706e0
npm verb etag "9SEU7JYB807P5YC1H5AGJ8DNR"
npm http request GET https://registry.npmjs.org/mongodb-prebuilt
npm http 304 https://registry.npmjs.org/mongodb-prebuilt
npm verb etag https://registry.npmjs.org/mongodb-prebuilt from cache
npm verb get saving mongodb-prebuilt to /root/.npm/registry.npmjs.org/mongodb-prebuilt/.cache.json
npm verb stack Error: No compatible version found: mongodb-prebuilt@'*'
npm verb stack Valid install targets:
npm verb stack ["3.0.6-6"]
npm verb stack 
npm verb stack     at installTargetsError (/opt/node-v0.12.6-linux-x64/lib/node_modules/npm/lib/cache/add-named.js:274:12)
npm verb stack     at next (/opt/node-v0.12.6-linux-x64/lib/node_modules/npm/lib/cache/add-named.js:253:17)
npm verb stack     at setData (/opt/node-v0.12.6-linux-x64/lib/node_modules/npm/lib/cache/add-named.js:230:5)
npm verb stack     at RES (/opt/node-v0.12.6-linux-x64/lib/node_modules/npm/node_modules/inflight/inflight.js:23:14)
npm verb stack     at f (/opt/node-v0.12.6-linux-x64/lib/node_modules/npm/node_modules/once/once.js:17:25)
npm verb stack     at fixName (/opt/node-v0.12.6-linux-x64/lib/node_modules/npm/lib/cache/add-named.js:29:5)
npm verb stack     at saved (/opt/node-v0.12.6-linux-x64/lib/node_modules/npm/lib/cache/caching-client.js:173:7)
npm verb stack     at FSReqWrap.oncomplete (fs.js:95:15)
npm verb cwd /home/rjurkov
npm ERR! Linux 3.16.0-39-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "mongodb-prebuilt" "--verbose"
npm ERR! node v0.12.6
npm ERR! npm  v2.11.2
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: mongodb-prebuilt@'*'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["3.0.6-6"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm verb exit [ 1, true ]

npm ERR! Please include the following file with any support request:
npm ERR!     /home/rjurkov/npm-debug.log
```

semver in this case is *, why is it not picking up a version with latest tag?

**Comment by KenanY** [comment]
Duplicate of https://github.com/npm/npm/issues/8855. This was fixed in [`npm@2.13.1`](https://github.com/npm/npm/releases/tag/v2.13.1).

**Comment by winfinit** [comment]
thank you
